README.md

```diff
@@ -95,6 +95,23 @@ Apart from Events and Commands Refund Plugin is also based on mechanisms derived
 
 Configuration of all elements mentioned above can be found and customized in `config.yml` file.
 
+## Payment requirements
+
+By default to refund your order you need to have at least one available payment method configured as offline gateway.
+In case your custom refund logic demands different type of gateway (for example stripe), you need to modify parameters
+used in `src/Resources/config/services/provider.xml`
+
+   ```xml
+        <parameters>
+                <parameter key="sylius_refund.supported_gateways" type="collection">
+                    <parameter>offline</parameter>
+                    <parameter>stripe</parameter>
+                </parameter>
+            </parameters>
+   ``` 
+
+After that you will probably want to process this payment, for that you should listen for an event for example `Sylius\RefundPlugin\Event\UnitsRefunded` 
```

Comment:
Online refund logic should be implemented if you need it. As the first try for the possible customization, you can check out `Sylius\RefundPlugin\Event\UnitsRefunded` event.

wdyt?